modules/rosapps/applications/devutils/shlextdbg/shlextdbg.cpp

```diff
@@ -683,6 +719,22 @@ int wmain(int argc, WCHAR **argv)
                         sei.fMask |= SEE_MASK_ASYNCOK ;
                     else if (isCmd(argc, argv, n, L"NOASYNC"))
                         sei.fMask |= SEE_MASK_NOASYNC;
+                    else if (isCmd(argc, argv, n, L"NOCONSOLE"))
+                        sei.fMask |= SEE_MASK_NO_CONSOLE;
+                    else if (isCmd(argc, argv, n, L"1000"))
+                        sei.fMask |= 0x00001000; // Unknown flag
+                    else if (isCmd(argc, argv, n, L"NOHOOKS"))
+                        sei.fMask |= 0x00002000;
+                    else if (isCmd(argc, argv, n, L"SITE"))
+                    {
+                        sei.fMask |= 0x08000000; // SEE_MASK_FLAG_HINST_IS_SITE
+                        sei.hInstApp = (HINSTANCE)GetLoggingServiceProvider();
+                    }
+                    else if (isCmd(argc, argv, n, L"FILL"))
+                    {
+                        sei.hInstApp = (HINSTANCE)0xCAFE;
+                        sei.hProcess = (HANDLE)0xDEADF00D;
```

Comment:
64-bit build fix:
```suggestion
                        sei.hInstApp = UlongToHandle(0xCAFE);
                        sei.hProcess = UlongToHandle(0xDEADF00D);
```
I don't think the hInstApp one needs an additional HINSTANCE handle... (otherwise it needs to be added).

Reply:
Yes, this should fix `error C4312: 'type cast': conversion from 'unsigned int' to 'HANDLE' of greater size`